docs/Information_nodes.rst

```diff
@@ -58,13 +58,20 @@ DGX A100
 .. _dgx_a100_nodes:
 
 DGX A100 nodes are NVIDIA appliances with 8 NVIDIA A100 Tensor Core GPUs. Each
-GPU has 40 GB of memory, for a total of 320 GB per appliance. The GPUs are
-interconnected via 6 NVSwitches which allows 4.8 TB/s bi-directional bandwidth.
+GPU has either 40GB or 80GB of memory, for a total of 320GB or 640GB per
+appliance. The GPUs are interconnected via 6 NVSwitches which allows 4.8 TB/s
+bi-directional bandwidth. See the table above for the specifications of each
```

Comment:
> ... which allow for 600 GB/s point-to-point bandwidth _(unidirectional)_ and a full bisection bandwidth of 4.8 TB/s _(bidirectional)_.

I don't like how NVIDIA selects the largest possible B/W number, which looks impressive but is misleading as to a single GPU's actual performance. For the regular user, knowing

1) The bandwidth of one GPU
2) That this bandwidth is available in both directions simultaneously (full-duplex)
3) That in a DGX-A100 there is full bisection bandwidth (all four pairs can communicate simultaneously at their full rate)

is probably enough.